pom.xml

```diff
@@ -1745,5 +1747,16 @@ under the License.
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>uniq-test-dirs</id>
+      <activation>
+        <property>
+          <name>uniq-test-dirs</name>
+        </property>
+      </activation>
+      <properties>
+        <accumulo.it.uniq.test.dir>true</accumulo.it.uniq.test.dir>
+      </properties>
+    </profile>
```

Comment:
I don't think we need to have all 3 of:
1. the profile `-Puniq-test-dirs`,
2. the property `-Daccumulo.it.uniq.test.dir=true`, and
3. the property that activates the profile that sets the property `-Duniq-test-dirs`

That's a bit excessive. Maintaining build profiles gets complicated, and it gets tricky to reason about their activation when there are so many levers. Usually, we try to only add new profiles when we need to manage multiple things, or at least something more than a single property.

I suggest removing the profile, and just keeping the single property.